util/timer.h

```diff
@@ -251,7 +252,7 @@ class Timer {
   Env* const env_;
   // This mutex controls both the heap_ and the map_. It needs to be held for
   // making any changes in them.
-  InstrumentedMutex mutex_;
+  mutable InstrumentedMutex mutex_;
```

Comment:
Sorry I missed the context. Why do we need a `mutable` here?

Reply:
It's to make `TEST_GetPendingTaskNum() const`, which needs to lock the mutex_.